### ISSUE TITLE
About AdaGCN

### ISSUE THREAD

**Issue by Tai-Xinwei** [Skyorca/OpenGDA#3]
Thank u for your excellent work! I have a question about the AdaGCN code. In AdaGCN, WGAN is used instead of the regular GAN. Therefore, should we remove the sigmoid function of discriminator?

**Comment by Skyorca** [comment]
You are right. You could refer to AdaGCN official repo in github, which is written in TensorFlow. If you implement a PyTorch version, let me know and maybe we could modify it together (and add you to collaborators)
